stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -213,6 +235,11 @@ func (m *Manager) UpdateTenantConfig(ctx context.Context, tu *TenantUpdate) (*Te
 		log.Ctx(ctx).Warnf("distribution account for tenant id %s updated to %s", tu.ID, *tu.DistributionAccount)
 	}
 
+	if tu.IsDefault != nil {
```

Comment:
I believe we should not update `IsDefault` in the same way as we update other fields.

When a `POST` or `PATCH` call attempts to update `is_default` we should consider doing the following: 
* **At the handler level**: 
   * Fail this call if `enable-default-tenant=false`. You need to enable default tenant feature to be able to update this flag. 
   * Only accept `true` as a valid value. We always want to have **one** valid default tenant. So if a user ever wants to turn off default for a tenant, it should turn it on for another tenant. 
* **In `UpdateTenantConfig`**: this call would set `is_default = true` for the current tenant, and would set `is_default = false` for all other tenants. 

wdyt ?

Reply:
That sounds good! I'll work on it.